test/unit/data/fake_names_index_template.json

```diff
@@ -0,0 +1,395 @@
+{
+  "order": 0,
+  "template": "names",
+  "settings": {},
+  "mappings": {
+    "_default_": {
+      "dynamic_templates": [
+        {
+          "string_fields": {
```

Comment:
This will catch all of your text/string fields, no reason to define them explicitly.